src/pages/Appointments/AppointmentsPage.tsx

```diff
@@ -87,6 +88,7 @@ import scheduleApis from "@/types/scheduling/scheduleApi";
 interface QueryParams {
   practitioner: string | null;
   slot: string | null;
+  page: number | null;
   date_from: string | null;
   date_to: string | null;
   search: string | null;
```

Comment:
is this interface even required anymore now that we are not using useQueryParams directly?